cljfmt/project.clj

```diff
@@ -21,5 +22,27 @@
                                  :optimizations :none}}}
               :test-commands
               {"dev" ["node" "target/out/tests.js"]}}
+  :native-image
+  {:name "cljfmt"
+   :opts
+   ["--verbose"
```

Comment:
As a minor formatting point, can you put the vector on the same line as `:opts`, e.g.

```clojure
:opts ["--verbose"
       "-H:+ReportExceptionStackTraces"
       ...]

Reply:
Sure